pkgs/os-specific/linux/tcmu-runner/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, ceph
+, cmake
+, fetchFromGitHub
+, glib
+, glusterfs
+, gperftools
+, kmod
+, libnl
+, pcre
+, pkg-config
+, util-linuxMinimal
+, zlib
+, cephSupport ? true
+, glusterSupport ? true
+, systemdSupport ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tcmu-runner";
+  version = "unstable-2022-02-21";
+
+  src = fetchFromGitHub {
+    owner = "open-iscsi";
+    repo = "tcmu-runner";
+    rev = "364ae611ffdd7c8fb3d1e733563b636bf3b44ba6";
+    hash = "sha256-SxAkpOzNB7/csEVfC6ptyyxJ8lu3YnRGiZhk4DsJnBA=";
+  };
+
+  postPatch = ''
+    # fix install paths, as they are currently hardcoded
+    # see also: https://github.com/open-iscsi/tcmu-runner/issues/683
+    substituteInPlace CMakeLists.txt \
+      --replace " /etc" " \''${CMAKE_INSTALL_PREFIX}/etc" \
+      --replace " /usr/" " \''${CMAKE_INSTALL_PREFIX}/"
+    for i in *.conf_install.cmake.in; do
+      substituteInPlace "$i" \
+        --replace " \"/etc" " \"\''${CMAKE_INSTALL_PREFIX}/etc"
+    done
+    for i in *.service; do
+      substituteInPlace "$i" \
+        --replace "=/usr/bin/" "=/$out/bin/"
```

Comment:
Why not to change it to the same `CMAKE_INSTALL_*`-style vars here too?

Even better: what about patch it and send the patch to upstream developers?

Reply:
> Why not to change it to the same `CMAKE_INSTALL_*`-style vars here too?

These files are not preprocessed by CMake, thus `CMAKE_INSTALL_*` style vars don't work there unless such preprocessing (utilizing `configure_file`) is additionally inserted.